test.sh

```diff
@@ -16,6 +16,32 @@ function main(){
     else
         run_production_tests;
     fi
+
+    # TODO: currently only work on development settings
```

Comment:
It has never occurred to me to use FIXME, which is muchmore sensible in this context and recognized even by PyCharm and other editors :+1: